### PR TITLE
init

### PR DESCRIPTION
**Pull request by ositer**
# 33 ACP-14 Django Dev

Copy django template, added project name in toml file, delete web dependencies in docker-compose.yaml , added excludes to gitignore, started for manual tests

## Type of change

Please delete options that are not relevant.

- [ x] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

# Checklist:

- [ x] My code follows the style guidelines of this project
- [ x] I have performed a self-review of my code

### CODE REVIEW

**Review by MWasile — CHANGES_REQUESTED** [review]
a quick correction is needed and meme must be added

File: newsletter/backend/.dockerignore [hunk 1]
```diff
@@ -0,0 +1,5 @@
+.idea
+.vscode
+*/**.pyc
```
Comment (BartekSzymik) [inline newsletter/backend/.dockerignore:3]: unify this with .gitignore **/*.pyc

**Review by BartekSzymik — CHANGES_REQUESTED** [review]
a few minor mistakes (accorfing Maciej W), rest is good

File: newsletter/backend/config/settings.py [hunk 2]
```diff
@@ -0,0 +1,206 @@
...
+    "ACCESS_TOKEN_LIFETIME": timedelta(minutes=5),
+    "REFRESH_TOKEN_LIFETIME": timedelta(days=25),
+    "ROTATE_REFRESH_TOKENS": True,
+    "BLACKLIST_AFTER_ROTATION": True,
+    # TODO: If updating last login in the table is desired, throttling must be set up in django rest framework
+    "UPDATE_LAST_LOGIN": False,
+    "ALGORITHM": "HS256",
+    "SIGNING_KEY": os.environ.get("JWT_SECRET_KEY", SECRET_KEY),
+    "VERIFYING_KEY": "",
+    "AUDIENCE": "azeno_quiz",
```
[the remark this answers is not in the document]
Comment (ositer) [reply newsletter/backend/config/settings.py]: I made new commit with changes.

File: newsletter/backend/.dockerignore (hunk 1 above)
Comment (ositer) [reply newsletter/backend/.dockerignore]: Done. I  made new commit with suggested changes.

**Review by MWasile — APPROVED** [review]
🆗

**Review by BartekSzymik — APPROVED** [review]
ok

**Pull request merged**